client/src/controllers/InitController.test.js

```diff
@@ -148,4 +148,39 @@ describe('InitController', () => {
       ]);
     });
   });
+
+  describe('with a detail', () => {
+    const handleEvent = jest.fn();
+    document.addEventListener('w-init:ready', handleEvent);
```

Comment:
```suggestion
    const handleEvent = jest.fn();

    document.addEventListener('w-init:ready', handleEvent);
    document.addEventListener('my-custom:event', handleEvent);

    const detail = { someMessage: 'some value' };
```

Just a small bit of clean up, let's keep our generic detail event up as a variable so it's easier to compare, also add an event listener for a new test below.

Reply:
Do I write another test to check that the custom event gets fired?

Reply:
Update: I just saw where you included it.